Name: Custom Carrier String
Version: 2.2.0-1
Author: Jason Robitaille
Description: Custom carrier string for the launcher.

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -388,7 +388,7 @@
 			if (this.phoneAppName)
 				Mojo.Dom.get('appname').innerHTML = this.phoneAppName;
 			else
-				Mojo.Dom.get('appname').innerHTML = this.carrierText;
+				Mojo.Dom.get('appname').innerHTML = payload.appmenu || payload.title;
 			this.phoneIsForeground = true;
 		}
 		else if (this.isPreferences(payload))  {
@@ -714,8 +714,8 @@
 				this.currentCarrierText = "Palm";
 			}
 			else {
-				this.carrierText = $L(payload.extended.networkName.escapeHTML());
-				this.currentCarrierText = payload.extended.networkName.escapeHTML();
+				this.carrierText = "carrier text here";  /*Edit this line*/
+				this.currentCarrierText = this.carrierText;
 			}
 			Mojo.Dom.get('carrier').innerHTML = this.carrierText;	   	
 			if(payload.extended.registration == 'home') {			
@@ -1133,8 +1133,8 @@
 					this.currentCarrierText = "Palm";
 				}
 				else {
-					this.carrierText = $L(networkMsg.networkName.escapeHTML());
-					this.currentCarrierText = networkMsg.networkName.escapeHTML();
+					this.carrierText = "carrier text here";  /*Edit this line*/
+					this.currentCarrierText = this.carrierText;
 				}
 				Mojo.Dom.get('carrier').innerHTML = this.carrierText;
 				this.simlocked = false;
